Code/Framework/AzToolsFramework/AzToolsFramework/Viewport/LocalViewBookmarkLoader.cpp

```diff
@@ -368,19 +481,28 @@ namespace AzToolsFramework
         }
 
         // if we managed to remove the bookmark
-        WriteBookmarksSettingsRegistryToFile();
+        WriteViewBookmarksSettingsRegistryToFile(localBookmarksFileName);
 
         return success;
     }
 
-    AZStd::optional<ViewBookmark> LocalViewBookmarkLoader::LoadLastKnownLocation() const
+    AZStd::optional<ViewBookmark> LocalViewBookmarkLoader::LoadLastKnownLocation()
     {
+        const LocalViewBookmarkComponent* bookmarkComponent = FindOrCreateLocalViewBookmarkComponent();
+        if (!bookmarkComponent)
+        {
+            AZ_Assert(bookmarkComponent, "Couldn't find or create LocalViewBookmarkComponent.");
```

Comment:
Same, `AZ_Error` instead.

Reply:
Missed this one sorry, I can do it later today or tomorrow (not at my main machine right now)